src/app/util/af-main-common.cpp

```diff
@@ -284,9 +282,9 @@ static EmberStatus send(EmberOutgoingMessageType type, uint64_t indexOrDestinati
     emAfApplyDisableDefaultResponse(&message[0]);
     emAfApplyRetryOverride(&apsFrame->options);
 
-    if (messageLength <= emberAfMaximumApsPayloadLength(type, indexOrDestination, apsFrame))
+    if (messageLength <= dmContext.GetExchangeContext().GetMTU())
```

Comment:
This is, unfortunately, not the right check.  I assume that `GetMTU` returns the size of the allowed CHIP application payload (though that's not what's implemented yet), but that payload will consist of the messageLength payload bytes here plus the framing that `chipSendUnicast` adds.  So we need to do something better here.  TODO with tracking issue is OK, and please tell me whether you plan to work on that  issue.

Reply:
Got it.